CRM/Contact/BAO/GroupNestingCache.php

```diff
@@ -69,7 +69,7 @@ static public function update() {
     }
 
     if (self::checkCyclicGraph($tree)) {
-      CRM_Core_Error::fatal(ts('We detected a cycle which we cant handle. aborting'));
+      CRM_Core_Error::fatal(ts("We detected a cycle which we can't handle. aborting"));
```

Comment:
Hey @mlutfy maybe we should just remove ts() from here. I don't even know what this string means in English!